src/util/system.cpp

```diff
@@ -285,6 +285,10 @@ bool ArgsManager::ParseParameters(int argc, const char* const argv[], std::strin
     m_settings.command_line_options.clear();
 
     for (int i = 1; i < argc; i++) {
+        if (argv[i] == nullptr) {
```

Comment:
I think this is the wrong place to fix things. In production this condition cannot occur, so this should be treated as a code error (which warrants documenting assumptions, or perhaps an assertion), not a user error (which warrants a runtime error message like this).

Reply:
An `assert` leaves the fuzzer with "deadly signal" which also breaks the build. Am not sure how to make it: 

* pass the fuzzer
* but without introducing any user-error checking logic

Reply:
And the fuzzer *should* get a deadly signal, because something in the code is wrong. It is correctly reporting an issue.

Reply:
Ok, then all is fine. 

However, the problem happened in an unrelated PR or mine (and will presumably happen in other people's PR's too). This was my actual motivation: how to get a clean build on all available travis platforms? :)

But, if it's acceptable to have a deadly signal, it's OK for me too.

Reply:
Well, if the fuzzer finds a bug, the bug should be fixed!

Reply:
Yes, that's true. Have updated the title of this PR and its description. 

Thanks to you and @elichai